packages/api/src/external/carequality/document/process-outbound-document-query-resps.ts

```diff
@@ -241,6 +241,9 @@ async function getRespWithDocsToDownload({
   requestId,
   response,
 }: OutboundDocQueryRespParam): Promise<DqRespWithDocRefsWithMetriportId[]> {
+  // TEMP #2349 - Remove console.log
+  const { log } = out(`TEMP #2349 docs to download - requestId ${requestId}, patient ${patientId}`);
```

Comment:
We mentioned request ID on Slack, but just to make sure we want the "jobId"'s `requestId`, not the HTTP's `reqId`.

The latter is automatically added by `out.log`